### PR TITLE
Add simplified samples for serving LLMs with Ray

### PR DESCRIPTION
**Pull request by andrewsykim**
## Description

This PR adds simplified samples for serving LLMs with Ray

## Tasks

* [X] The [contributing guide](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/.github/CONTRIBUTING.md) has been read and followed.
* [X] The samples added / modified have been fully tested.
* [X] Workflow files have been added / modified, if applicable.
* [X] Region tags have been properly added, if new samples.
* [X] All dependencies are set to up-to-date versions, as applicable.

### CODE REVIEW

File: ai-ml/gke-ray/rayserve/llm/gemma-2b-it/ray-service.yaml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      route_prefix: /
+      import_path:  ai-ml.gke-ray.rayserve.llm.serve:model
+      deployments:
+      - name: VLLMDeployment
+        num_replicas: 1
+        ray_actor_options:
+          num_cpus: 8
+          num_gpus: 1
+      runtime_env:
+        working_dir: "https://github.com/andrewsykim/kubernetes-engine-samples/archive/ray-llms.zip"
```
Comment (andrewsykim) [inline ai-ml/gke-ray/rayserve/llm/gemma-2b-it/ray-service.yaml:33]: NOTE: reference to my develop branch is temporary for testing

**Comment by bourgeoisor** [comment]
@andrewsykim lemme know when that PR is good for review

**Comment by andrewsykim** [comment]
> @andrewsykim lemme know when that PR is good for review

Feel free to review it now

**Review by bourgeoisor — APPROVED** [review]
LGTM

**Pull request merged**